### PR TITLE
Ecp 1160 data cleardown

### PR DESCRIPTION
**Pull request by and-lucas-kelly**

### CODE REVIEW

File: database_scripts/prod_data_backup.sql [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+ alter table manual_claim_clear_down
+   add primary key (claim_id, eligibility_id);
+
+insert into manual_claim_clear_down (claim_id, eligibility_id) 
+select c.id "claim_id",
+	   a.id "eligibility_id"
+  from public.student_loans_eligibilities a,
+  	   public.claims c
+ where a.id = c.eligibility_id
+   and a.created_at BETWEEN '2021-04-01T00:00:01' and '2021-10-29T14:33:14';
```
Comment (WorkSutton) [inline database_scripts/prod_data_backup.sql:19]: @and-lucas-kelly any reason this is not 31-August-2021?

File: database_scripts/prod_data_investigation.sql [hunk 2]
```diff
@@ -0,0 +1,98 @@
+--View a list of submitted ECP claims' eligibility criteria
+select * 
+  from public.early_career_payments_eligibilities
+ where created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14' -- e.g. Datetime format 2021-04-29T14:33:14
```
Comment (WorkSutton) [inline database_scripts/prod_data_investigation.sql:4]: @and-lucas-kelly same question as above about why date is 29th August and not 31st? Feels to me as if 31st is safer as its the end of the previous AcademicYear

File: database_scripts/prod_data_investigation.sql [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+ order by count(*);
+
+select * 
+  from public.claims 
+ where created_at BETWEEN '2021-04-01T00:00:00' and '2021-05-31T23:59:59' -- e.g. Datetime format 2021-04-29T14:33:14
+ 
+
+select * 
+  from public.claims 
+ where created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14' -- e.g. Datetime format 2021-04-29T14:33:14
```
Comment (WorkSutton) [inline database_scripts/prod_data_investigation.sql:32]: @and-lucas-kelly the date of 29th October 2021? What does that relate to? I was expecting to see 29th August 2021 if its same as reason for that date in my previous questions.

File: database_scripts/prod_data_investigation.sql [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+
+--View a list of submitted ECP claims' eligibility criteria
+select c.first_name, 
+	   c.surname, 
+	   a.id "ecp-id", 
+	   c.id "claim-id"
+  from public.early_career_payments_eligibilities a,
+  	   public.claims c
+where a.id = c.eligibility_id
+  and a.created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14'; -- e.g. Datetime format 2021-04-29T14:33:14
```
Comment (WorkSutton) [inline database_scripts/prod_data_investigation.sql:53]: @and-lucas-kelly same queston on October date as above. I think I have missed something on these date ranges. Apologies if the question is silly.

File: database_scripts/prod_data_investigation.sql [hunk 5]
```diff
@@ -0,0 +1,98 @@
...
+
+--View a list of submitted ECP claims' eligibility criteria
+select c.first_name, 
+	   c.surname, 
+	   a.id "sl-id", 
+	   c.id "claim-id"
+  from public.student_loans_eligibilities a,
+  	   public.claims c
+where a.id = c.eligibility_id
+  and a.created_at BETWEEN '2021-06-01T00:00:01' and '2021-10-29T14:33:14'; -- e.g. Datetime format 2021-04-29T14:33:14
```
Comment (WorkSutton) [inline database_scripts/prod_data_investigation.sql:63]: @and-lucas-kelly same question as previously

File: database_scripts/prod_data_investigation.sql [hunk 6]
```diff
@@ -0,0 +1,98 @@
...
+--	   public.amendments a
+--	   public.decisions d
+public.tasks t
+ where ecpe.id = c.eligibility_id
+--   and st.claim_id = c.id
+--   and n.claim_id = c.id
+--   and a.claim_id = c.id
+--   and d.claim_id = c.id
+   and t.claim_id = c.id
+   and ecpe.created_at BETWEEN '2021-06-08T00:00:01' and '2021-10-29T14:33:14'; -- e.g. Datetime format 2021-04-29T14:33:14
```
Comment (WorkSutton) [inline database_scripts/prod_data_investigation.sql:92]: @and-lucas-kelly same question as previously

File: database_scripts/prod_data_restore.sql [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+select * 
+  from early_career_payments_eligibilities_bkp;
+  
+insert into public.claims
+select * 
+  from claims_bkp;
+  
+insert into public.student_loans_eligibilities  
+select * 
+  from student_loans_eligibilities_bkp;
```
Comment (WorkSutton) [inline database_scripts/prod_data_restore.sql:19]: @and-lucas-kelly could we add the extra blank line (CR) please

**Review by WorkSutton — COMMENTED** [review]
Just some date questions

File: database_scripts/prod_data_backup.sql (hunk 1 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_backup.sql:19]: updated to end of first week in September as that is the last possible date for Private beta

File: database_scripts/prod_data_investigation.sql (hunk 2 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_investigation.sql:4]: this is purely the investigation script to identify data

File: database_scripts/prod_data_investigation.sql (hunk 3 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_investigation.sql:32]: this is purely the investigation script to identify data

File: database_scripts/prod_data_investigation.sql (hunk 4 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_investigation.sql:53]: this is purely the investigation script to identify data

File: database_scripts/prod_data_investigation.sql (hunk 5 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_investigation.sql:63]: this is purely the investigation script to identify data

File: database_scripts/prod_data_investigation.sql (hunk 6 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_investigation.sql:92]: this is purely the investigation script to identify data

File: database_scripts/prod_data_restore.sql (hunk 7 above)
Comment (and-lucas-kelly) [reply database_scripts/prod_data_restore.sql:19]: added now

**Comment by WorkSutton** [comment]
@and-lucas-kelly thanks for the call and the clarification.

**Pull request merged**